tools/pipeline/Main.cpp

```diff
@@ -137,7 +147,7 @@ static void runPipeline(Runner &Pipeline) {
   for (const auto &Target : Targets)
     AbortOnError(parseTarget(ToProduce, Target, Registry));
 
-  auto *Stream = Silence ? nullptr : &dbgs();
+  auto *Stream = (Verbose or not Silence) ? &outs() : nullptr;
```

Comment:
* The way you're mixing up verbose and silence is confusing. The tool by default should be silent, and in case of verbose it should print extra stuff on `stderr`. I really think you should be using `Logger`, `revng-pipeline` is acting more and more as a special kid in this sense.
* Also, you should print on `stdout` only stuff that makes sense to save on a file and has a well-defined format.

Reply:
fixed, toh, outs() supports color coded stuff, dbgs does not